src/app/pages/HomePage/Categories/component.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Row, Col } from 'react-bootstrap';
+import { useTranslation } from 'react-i18next';
+import Category from 'shared/components/Category';
+import { AllProjectsIcon, ArtIcon, ChildrenIcon, CovidIcon, EcoIcon, EducationIcon } from 'shared/images';
+
+const Categories: React.FC = () => {
+  const { t } = useTranslation();
+
+  return (
+    <Row>
+      <Col md={4}>
+        <Category title={t('EDUCATION')} count={t('PROJECTS', { count: 43 })} icon={EducationIcon} />
```

Comment:
please, keep these values in `categories` array, like `menuItems`, and then map items

```js
categories.map(category => {
 <Col md={4}>
     <Category 
        title={category.title} 
        count={t('PROJECTS', { count: category.count })} 
        icon={category.icon} 
        link={category.link}
      />
  </Col>
})

```

Reply:
Done